packages/fiori/src/themes/NotificationListGroupItem.css

```diff
@@ -28,7 +28,7 @@
 }
 
 .ui5-nli-group-toggle-btn {
-	margin-right: 1rem;
+	margin-right: 0.75rem;
```

Comment:
margin-right: 1.25rem;
	margin-left: 0.5rem;

Reply:
Thanks for the feedback, Teo.

 Just to clarify, in [openui5](https://openui5nightly.hana.ondemand.com/entity/sap.m.NotificationListGroup/sample/sap.m.sample.NotificationListGroup) the toggle button has **margin-right: 0.75rem** 
```css
.sapMNLGroupCollapseButton {
       margin-right: 0.75rem;
       min-width: 2.25rem;
       max-width: 2.25rem;
}
```

When I apply **margin-right: 0.75 rem** the result is: (seems aligned to me)
<img width="246" alt="Screenshot 2020-12-18 at 5 16 54 PM" src="https://user-images.githubusercontent.com/15702139/102630874-c189ef00-4155-11eb-926d-7d473407befe.png">

When I apply **margin-right: 1.25rem; and margin-left: 0.5rem;** the result is (not quite  aligned):
<img width="237" alt="Screenshot 2020-12-18 at 5 24 52 PM" src="https://user-images.githubusercontent.com/15702139/102631015-f433e780-4155-11eb-98d4-81bcefbbb3ac.png">